lib/linguist/heuristics.yml

```
@@ -217,6 +230,8 @@ disambiguations:
   - language: Slice
 - extensions: ['.inc']
   rules:
+  - language: FreeBasic
+    pattern: '^\s*#(define|endif|endmacro|if|ifdef|ifndef|include|lang)\s'
```

This is slightly more performant:

```suggestion
    pattern: '^[ \t]*#(?:define|endif|endmacro|ifn?def|if|include|lang)\s'
```